### ISSUE TITLE
Go to the rain training problem

### ISSUE THREAD

**Issue by Qixin-oss** [Algolzw/image-restoration-sde#121]
Why do I train the rain removal model on 4090 and the psnr can only reach about 28?

![图片1](https://github.com/user-attachments/assets/4b32c7db-d0ca-401a-bae7-d92d690922dc)
![图片2](https://github.com/user-attachments/assets/92e5dc33-4b94-4a0a-95ff-cf84c27659b7)

**Comment by Algolzw** [comment]
We use PSNR_Y and SSIM_Y as the evaluation metrics. But the results should be higher (at least higher than 30 in terms of PSNR_Y) on the Rain100H dataset. Can you share your settings such that I can find the difference?

**Comment by Qixin-oss** [comment]
#### general settings
name: ir-sde
use_tb_logger: true
model: denoising
distortion: derain
gpu_ids: [0]

sde:
  max_sigma: 10
  T: 100
  schedule: cosine # linear, cosine
  eps: 0.005

degradation: # for some synthetic dataset that only have GTs
  # for denoising
  sigma: 25
  noise_type: G # Gaussian noise: G

  # for super-resolution
  scale: 4
  
#### datasets
datasets:
  train:
    name: Train_Dataset
    mode: LQGT
    dataroot_GT: dataset/rain/trainH/GT
    dataroot_LQ: dataset/rain/trainH/LQ

    use_shuffle: true
    n_workers: 4  # per GPU
    batch_size: 4
    GT_size: 128
    LR_size: 128
    use_flip: true
    use_rot: true
    color: RGB
  val:
    name: Val_Dataset
    mode: LQGT
    dataroot_GT: dataset/rain/testH/GT
    dataroot_LQ: dataset/rain/testH/LQ


#### network structures
network_G:
  which_model_G: ConditionalUNet
  setting:
    in_nc: 3
    out_nc: 3
    nf: 64
    depth: 4

#### path
path:
  pretrain_model_G: ~
  strict_load: true
  resume_state: ~

#### training settings: learning rate scheme, loss
train:
  optimizer: Adam # Adam, AdamW, Lion
  lr_G: !!float 1e-4
  lr_scheme: MultiStepLR
  beta1: 0.9
  beta2: 0.99
  niter: 700000
  warmup_iter: -1  # no warm up
  lr_steps: [200000, 400000, 600000]
  lr_gamma: 0.5
  eta_min: !!float 1e-7

  # criterion
  is_weighted: False
  loss_type: l1
  weight: 1.0

  manual_seed: 0
  val_freq: !!float 5e3

#### logger
logger:
  print_freq: 100
  save_checkpoint_freq: !!float 5e3
In the settings of the IR-SDE
